minutes/2021/04/27-fe-initiatives-minutes.md

```diff
@@ -0,0 +1,69 @@
+When: April 27, 2021 @ 15:00 CET
+Where: Online
+Attending: Frontend and UX People...
+
+# Agenda
+
+## General updates
+
+### Add the shared calendars
+
+Feel free to join meetings you are interested in, or check the notes if
+you just want to see what was discussed and potentially decided.
+
+-   Frontend Group: `c_aebpolkmfvanjcfgf8f96bnbi8@group.calendar.google.com`
+-   Release calendar: `dhis2.org_vhltgg78od75pqm4m39t7arvc8@group.calendar.google.com`
+-   Core dev calendar: `dhis2.org_tbep59bdcpc5k0rl2c8ctp962k@group.calendar.google.com`
+-   Dev holiday calendar: `dhis2.org_a5jeon529ng4ad8srv3ajlnp0s@group.calendar.google.com`
+
+## Recruitment
+
+Who wants to help with recruitment? Reach out to Viktor if you want to
+help out with who will be your future co-workers.
+
+## Working groups
+
+### GQL => App Runtime Queries
+
+https://jira.dhis2.org/browse/LIBS-158
+
+Opt-in variant to use GraphQL queries instead of our own Query object
+notation.
+
+Some benefits include, better syntax highlighting and static query
+analysis for validation etc.
+
+### UI#463 / DataTable
+
+There is a clear way forward, so should be able to be merged soon. Maybe
+even as soon as next week.
+
+### Cypress / Network Shim
+
+It's been a roller coaster, but the network shim is working !
```

Comment:
It seems to be working*
PR still has to be tested, reviewed & approved (who knows what quirks we'll encounter..)

Reply:
I have a quote from @HendrikThePendric that says it works so I'm choosing to believe. :dango: